zulipterminal/api_types.py

```diff
@@ -53,7 +55,7 @@ class Message(TypedDict, total=False):
     avatar_url: str
     content_type: str
     match_content: str  # If keyword search specified in narrow params.
-    match_subject: str  # If keyword search specified in narrow params.
+    match_topic: str  # If keyword search specified in narrow params.
```

Comment:
Do you have support for this migration?

Reply:
No. Actually, as `subject` was changed to `topic`, I felt it would be good to rename this accordingly. But as the API is still the same, I'll retain the original names for this and `orig_topic`